Dockerfile

```diff
@@ -1,11 +1,24 @@
 # Arguments
-ARG NODE_VERSION=16.15.0-alpine
+ARG NODE_VERSION=lts-alpine
```

Comment:
If we don't then the docker build fails, and it will fail with a silly error that is not obvious to the missing ARG.

Unfortunately, there is no way to mark an ARG as required, so the error you get is really not useful. Thus why I decided to rather default it to LTS, and rather add a comment in the Dockerfile, and the main README.